lib/StyleCop/StyleCopCheck.proj

```diff
@@ -0,0 +1,34 @@
+<Project DefaultTargets="Full" xmlns= "http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+
+    <UsingTask AssemblyFile="StyleCop.dll" TaskName="StyleCopTask"/>
+
+    <PropertyGroup>
+        <FolderToAnalyse>..\..\src</FolderToAnalyse>

```

Comment:
I am not clear on what you mean here... At the minute, I was calling `msbuild .\StyleCopCheck.proj` directly from the command line, wasn't sure how to add it into nant, that is where I was hoping you would come in :-)